WalletLibrary/WalletLibraryTests/Requests/Processors/OpenIdRequestProcessorTests.swift

```diff
@@ -83,6 +83,108 @@ class OpenIdRequestProcessorTests: XCTestCase {
         }
     }
     
+    func testProcessPresentationRequest_WithOneExtension_ReturnsVerifiedIdRequest() async throws {
+        
+        // Arrange
+        let expectedStyle = MockRequesterStyle(requester: "mock requester")
+        let expectedRequirement = MockRequirement(id: "mockRequirement324")
+        let expectedRootOfTrust = RootOfTrust(verified: true, source: "mock source")
+        let expectedContent = PresentationRequestContent(style: expectedStyle,
+                                                         requirement: expectedRequirement,
+                                                         rootOfTrust: expectedRootOfTrust,
+                                                         requestState: "mock state",
+                                                         callbackUrl: URL(string: "https://test.com")!)
```

Comment:
Can we use a domain we control or a non-existent domain?

Reply:
yes changed.